lib/ex_force.ex

```diff
@@ -169,6 +169,20 @@ defmodule ExForce do
   end
 
   @doc """
+  Creates a SObject.
+
+  See [SObject Rows](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_sobject_retrieve.htm)
```

Comment:
The actual doc is at https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_sobject_basic_info.htm

Reply:
ah yea , forgot to updated it.